app/src/main/kotlin/at/bitfire/davdroid/ui/account/WebcalFragment.kt

```diff
@@ -60,6 +60,32 @@ class WebcalFragment: CollectionsFragment() {
         }
     }
 
+    private val menuProvider = object : CollectionsMenuProvider() {
+        override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
+            menuInflater.inflate(R.menu.caldav_actions, menu)
+        }
+
+        override fun onPrepareMenu(menu: Menu) {
+            super.onPrepareMenu(menu)
+            menu.findItem(R.id.create_calendar).isVisible = false
+        }
+
+        override fun onMenuItemSelected(menuItem: MenuItem): Boolean {
+            if (super.onMenuItemSelected(menuItem)) {
+                return true
+            }
+
+            if (menuItem.itemId == R.id.create_calendar) {
+                val intent = Intent(requireActivity(), CreateCalendarActivity::class.java)
+                intent.putExtra(CreateCalendarActivity.EXTRA_ACCOUNT, accountModel.account)
+                startActivity(intent)
+                return true
+            }
+
+            return false
+        }
```

Comment:
@ArnyminerZ By the way this code was erroneously added. I have removed it in d79e737f2bc377a04efd548f4c22f3c5bc194457